husky_control/config/control_left_ur5.yaml

```diff
@@ -0,0 +1,22 @@
+l_arm_controller:
```

Comment:
On the real robot, the intent of this is to be used for moveit's controller manager